test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -277,6 +303,11 @@ var _ = Describe("Seed controller tests", func() {
 							g.Expect(testClient.Delete(ctx, mutatingWebhookConfiguration)).To(Succeed())
 							g.Expect(testClient.Delete(ctx, validatingWebhookConfiguration)).To(Succeed())
 						}).Should(Succeed())
+
+						Eventually(func(g Gomega) {
```

Comment:
Why did you add this `Eventually` block?

Reply:
Sometimes, I saw that the webhookConfig is not deleted yet and the apply of resources fail because the webhook endpoints are not served.

Reply:
Ok got it. However, the `Eventually` block will not help solve this case. 
It doesn't matter if the manager's client has observed that the webhooks have been deleted. It only matters whether the API server's informers in the admission plugins have observed the deletion, and you can't wait for that / check that in the test (there is simply no way). So intermittent failures are always expected in such cases.